analyzers/src/SonarAnalyzer.ShimLayer.CodeGeneration/SyntaxLightupGenerator.cs

```diff
@@ -1238,7 +1238,9 @@ private sealed class NodeData
                 nameof(LambdaExpressionSyntax),
                 nameof(ParenthesizedLambdaExpressionSyntax),
                 nameof(SimpleLambdaExpressionSyntax),
-                nameof(ClassDeclarationSyntax));
+                nameof(ClassDeclarationSyntax),
+                nameof(StructDeclarationSyntax),
+                nameof(TypeDeclarationSyntax));
```

Comment:
TypeDeclarationSyntax must be removed. See here why:
https://github.com/SonarSource/sonar-dotnet/pull/8209/files#diff-7e73935ebcefdb4899aac180afd234c25d4fdc1adff4a2db8391ba7cb59622b0R1241-R1244